pkg/apis/pipeline/v1alpha1/pipeline_types.go

```diff
@@ -152,6 +152,12 @@ func (pt PipelineTask) Deps() []string {
 			deps = append(deps, rd.From...)
 		}
 	}
+	// Add any dependents declared via workspace from clauses
```

Comment:
this isnt your doing but it's not clear to me what this `Deps` function is for exactly - any chance you'd add a docstring for it? 🙏  no worries if not